### PR TITLE
Improvements to find and replace in source view

### PR DESCRIPTION
**Pull request by NatashaWso2**
Fixes https://github.com/ballerinalang/composer/issues/2085

Before : 
before-customizing

After : 
find

word-highlight

### CODE REVIEW

File: modules/web/js/ballerina/utils/ace-styles.css [hunk 1]
```diff
@@ -10,3 +10,63 @@
 .text-editor.ace_editor {
     font: 12px/normal 'Droid Sans Mono', monospace;
 }
+
```
Comment (nadeeshaan) [inline modules/web/js/ballerina/utils/ace-styles.css]: Shall we add a comment specifically mentioning what the use case we are addressing here

File: modules/web/js/ballerina/utils/ace-styles.css [hunk 2]
```diff
@@ -10,3 +10,63 @@
...
+.bal-source-editor .ace_search {
+    background-color: #707276 !important;
+    border-color: #707276 !important;
+    padding-left: 13px;
+    padding-bottom: 10px;
+    border-radius: 5px 5px 5px 5px !important;
```
Comment (hemikak) [inline modules/web/js/ballerina/utils/ace-styles.css]: Can use border-radius: 5px !important; here.

File: modules/web/js/ballerina/utils/ace-styles.css [hunk 3]
```diff
@@ -10,3 +10,63 @@
...
+    background-color: #949994;
+}
+
+.bal-source-editor .ace_search .ace_search_options{
+    margin-right: 30px;
+    position: absolute;
+    right: 0;
+}
+
+.bal-source-editor .ace_search .ace_searchbtn{
```
Comment (hemikak) [inline modules/web/js/ballerina/utils/ace-styles.css]: Need spacing before {

File: modules/web/js/ballerina/utils/ace-styles.css [hunk 4]
```diff
@@ -10,3 +10,63 @@
...
+    background-color: #949597;
+    width: 40px !important;
+}
+
+.bal-source-editor .ace_search .ace_searchbtn.next{
```
Comment (hemikak) [inline modules/web/js/ballerina/utils/ace-styles.css]: Need spacing before {

File: modules/web/js/ballerina/utils/ace-styles.css [hunk 5]
```diff
@@ -10,3 +10,63 @@
...
+    background-color: #949597;
+    width: 26px !important;
+}
+
+.bal-source-editor .ace_search .ace_searchbtn.prev{
```
Comment (hemikak) [inline modules/web/js/ballerina/utils/ace-styles.css]: Need spacing before {

File: modules/web/js/ballerina/utils/ace-styles.css [hunk 6]
```diff
@@ -10,3 +10,63 @@
...
+    position: absolute;
+    right: 0;
+}
+
+.bal-source-editor .ace_search .ace_searchbtn{
+    background-color: #949597;
+    width: 40px;
+}
+
+.bal-source-editor .ace_search .ace_searchbtn.next{
```
Comment (hemikak) [inline modules/web/js/ballerina/utils/ace-styles.css]: Need spacing before {

File: modules/web/js/ballerina/utils/ace-styles.css (hunk 2 above)
Comment (NatashaWso2) [reply modules/web/js/ballerina/utils/ace-styles.css]: Fixed with 8ec670a0e740d6a43805b3744902cdbe5fe80046

File: modules/web/js/ballerina/utils/ace-styles.css [hunk 7]
```diff
@@ -10,3 +10,63 @@
...
+    background-color: #949597;
+    width: 26px;
+}
+
+.bal-source-editor .ace_search .ace_searchbtn.prev{
```
Comment (hemikak) [inline modules/web/js/ballerina/utils/ace-styles.css]: Need spacing before {

File: modules/web/js/ballerina/utils/ace-styles.css [hunk 8]
```diff
@@ -10,3 +10,63 @@
...
+.ace_editor .ace_marker-layer .ace_selected-word {
+    background: transparent;
+    border: 1px solid #ffefd5;
+}
+
+.bal-source-editor .ace_search .ace_searchbtn_close {
+    background-color: #949994;
+}
+
+.bal-source-editor .ace_search .ace_search_options{
```
Comment (hemikak) [inline modules/web/js/ballerina/utils/ace-styles.css]: Need spacing before {

File: modules/web/js/ballerina/utils/ace-styles.css [hunk 9]
```diff
@@ -10,3 +10,63 @@
...
+.bal-source-editor .ace_search {
+    background-color: #707276;
+    border-color: #707276;
+    padding-left: 13px;
+    padding-bottom: 10px;
+    border-radius: 5px;
+    max-width: 355px;
+}
+
+.bal-source-editor .ace_search_options .ace_button.checked{
```
Comment (hemikak) [inline modules/web/js/ballerina/utils/ace-styles.css]: Need spacing before {

Comment (NatashaWso2) [reply modules/web/js/ballerina/utils/ace-styles.css]: Fixed with 687c6ed0f0d73e65dbab8c3f6f578393e81f44b8

File: modules/web/js/ballerina/utils/ace-styles.css (hunk 8 above)
Comment (NatashaWso2) [reply modules/web/js/ballerina/utils/ace-styles.css]: Fixed with 687c6ed0f0d73e65dbab8c3f6f578393e81f44b8

File: modules/web/js/ballerina/utils/ace-styles.css (hunk 3 above)
Comment (NatashaWso2) [reply modules/web/js/ballerina/utils/ace-styles.css]: Fixed with 687c6ed0f0d73e65dbab8c3f6f578393e81f44b8

File: modules/web/js/ballerina/utils/ace-styles.css (hunk 4 above)
Comment (NatashaWso2) [reply modules/web/js/ballerina/utils/ace-styles.css]: Fixed with 687c6ed0f0d73e65dbab8c3f6f578393e81f44b8

File: modules/web/js/ballerina/utils/ace-styles.css (hunk 6 above)
Comment (NatashaWso2) [reply modules/web/js/ballerina/utils/ace-styles.css]: Fixed with 687c6ed0f0d73e65dbab8c3f6f578393e81f44b8

File: modules/web/js/ballerina/utils/ace-styles.css (hunk 7 above)
Comment (NatashaWso2) [reply modules/web/js/ballerina/utils/ace-styles.css]: Fixed with 687c6ed0f0d73e65dbab8c3f6f578393e81f44b8

File: modules/web/js/ballerina/utils/ace-styles.css (hunk 5 above)
Comment (NatashaWso2) [reply modules/web/js/ballerina/utils/ace-styles.css]: Fixed with 687c6ed

**Pull request merged**